clientgui/BOINCGUIApp.cpp

```diff
@@ -599,7 +599,7 @@ void CBOINCGUIApp::OnInitCmdLine(wxCmdLineParser &parser) {
 #if (defined(__WXMAC__) && defined(_DEBUG))
     parser.AddLongOption("NSDocumentRevisionsDebugMode", _("Not used: workaround for bug in XCode 4.2"));
 #endif
-    parser.AddSwitch("nd", "no-daemon", _("Not run the daemon"));
+    parser.AddSwitch("nd", "no-daemon", _("Don't run the client"));
```

Comment:
You could change the name of the command line switch too.

Short options are typically one character after hyphen. wxWidgets' command line parser seems to handle `-nd` without confusing it with `-n` or `-d` but I'm not sure if that's by design or by accident. I'd remove the short option.

`--no-daemon` is with hyphen but the rest of Manager and client uses underscore. Please change that.